docs/reference/Admin_Actions.md

```diff
@@ -4,6 +4,56 @@ When you consider that most backend services are a black box of code and decisio
 
 With that said this document will serve as the ongoing history of administrative actions that must be taken against the backend.
 
+## 2024 - January 30
+
+### linter-shellcheck
+
+The community package `linter-shellcheck` has been discovered to be broken. The source code contains an [error](https://github.com/AtomLinter/linter-shellcheck/issues/160). This package's source code has since been archived as of August 7, 2023. So it is now impossible for this package to ever receive an update to resolve it's issue.
+
+But the package has been forked to the [`pulsar-cooperative`](https://github.com/pulsar-cooperative) organization. This forked package is now recommended to download.
+
+As this package is now permanently broken, and archived, it'll receive both the [`Archived`](./badge-spec.md#archived) and [`Broken`](./badge-spec.md#broken), as well as [`Deprecated`](./badge-spec.md#deprecated) badges to reflect it's status, with installation of [`linter-shellcheck-pulsar`](https://web.pulsar-edit.dev/packages/linter-shellcheck-pulsar) instead recommended.
+
+### linter-remark
+
+The community package `linter-remark` has been discovered to be broken. The source code contains an [error](https://github.com/pulsar-edit/package-backend/issues/212). This package's source code has since been archived as of June 18, 2022. So it is now impossible for this package to ever receive an update to resolve it's issue.
+
+As this package is now permanently broken, and archived, it'll receive both the [`Archived`](./badge-spec.md#archived) and [`Broken`](./badge-spec.md#broken) badges to reflect it's status.
+
+### atom-ternjs
+
+The community package [`atom-ternjs`](https://web.pulsar-edit.dev/packages/atom-ternjs) has been reported to be broken.
+
+This package was the first to be ported the [`pulsar-cooperative`](https://github.com/pulsar-cooperative) organization, in the effort to help alleviate the ownership burden of broken community packages.
+
+This forked package is now recommended to download.
+
+So this package will receive the [`Deprecated`](./badge-spec.md#deprecated) badge, with installation of [`pulsar-ternjs`](https://web.pulsar-edit.dev/packages/pulsar-ternjs) instead recommended.
+
+### compare-files
+
+The community package `compare-files` has been discovered to be broken. The source code contains an [error](https://github.com/atom-compare-files/atom-compare-files/issues) that was originally reported before Pulsar had ever been created. This package's source code has since been archived as of January 21, 2019. So it is now impossible for this package to ever receive an update to resolve it's issue.
```

Comment:
```suggestion
The community package `compare-files` has been discovered to be broken. The source code contains an [error](https://github.com/atom-compare-files/atom-compare-files/issues) that was originally reported before Pulsar had ever been created. This package's source code has since been archived as of January 21, 2019. So it is now impossible for this package to ever receive an update to resolve its issue.
```